oauth2-example-app/src/test/java/com/clouway/oauth2/exampleapp/storage/InMemoryClientRepositoryTest.java

```diff
@@ -8,10 +8,8 @@
  */
 public class InMemoryClientRepositoryTest extends ClientRepositoryContractTest {
 
-  private InMemoryClientRepository repository = new InMemoryClientRepository();
-
   @Override
-  public ClientRepository createRepository() {
-    return repository;
+  public ClientRepository createRepository(String defaultClientId) {
```

Comment:
This signature doesn't look good. 

What is the goal of the defaultClientId ? Client is represented minimum with clientId and secret.